src/functionalTest/java/uk/gov/hmcts/reform/waworkflowapi/controllers/CreateTaskTest.java

```diff
@@ -201,7 +205,7 @@ public void transition_create_overdue_task() {
             .then()
             .statusCode(HttpStatus.CREATED_201);
 
-        await().ignoreException(AssertionError.class).pollInterval(1, SECONDS).atMost(60, SECONDS).until(
+        await().ignoreException(AssertionError.class).pollInterval(1, SECONDS).atMost(20, SECONDS).until(
```

Comment:
Interesting I thought the assertion always happened after receiving the response.
I guess you were running into some specific issues I'll keep in mind for the future :)  👍